skbio/stats/ordination/_utils.py

```diff
@@ -196,3 +196,86 @@ def f_matrix(E_matrix):
     col_means = E_matrix.mean(axis=0, keepdims=True)
     matrix_mean = E_matrix.mean()
     return E_matrix - row_means - col_means + matrix_mean
+
+
```

Comment:
These existed as utility methods before, making them private now might break others' code.

Reply:
As long as they were never exported from _utils.py there isn't a contract. We would consider a "public function" in a "private package" to be package private (usable within skbio only). That being said, I'm relatively unfamiliar with this section of the codebase, so that "rule" may not really apply here.

Reply:
@ebolyen and @ElDeveloper I tried making the f_matrix and e_matrix (and f_matrix_inplace and e_matrix_inplace) methods private, but when I removed the references from skbio.ordination.__init__.py 

```
# I tried removing the f/e matrix method names and import from below
from ._utils import (mean_and_std, scale, svd_rank, corr, e_matrix, f_matrix,
                     center_distance_matrix, _e_matrix_inplace,
                     _f_matrix_inplace)

__all__ = ['ca', 'rda', 'cca', 'pcoa', 'pcoa_biplot', 'OrdinationResults',
           'mean_and_std', 'scale', 'svd_rank', 'corr',
           'e_matrix', 'f_matrix', 'center_distance_matrix',
           '_e_matrix_inplace', '_f_matrix_inplace']
```

Reply:
@ElDeveloper ah, thanks for clarifying, you're right, you can import directly from the utils file. I've made those methods private now and removed the references from the `__init__.py`